guides/release/code-editors/index.md

```diff
@@ -7,11 +7,9 @@ many of which are created and maintained by the developer community:
 Visual Studio Code is a code editor optimized for building and debugging modern web applications. 
 Visual Studio Code is one of the most popular text editors among Ember developers.
 
-[Ember Language Server](https://marketplace.visualstudio.com/items?itemName=EmberTooling.vscode-ember) -
```

Comment:
I moved Ember Language Server to the bottom, because the last update was in 2020 -- this may come off insensitive to the folks who maintain the project (thank you for your service, btw, everything has to start somewhere!), but I believe the true language server right now _is_ the uELS. (The one on @lifeart's GitHub).
It'd be great if we could promote the uELS to just ELS in the vs code marketplace.. but.. #politics.